.env.example

```
@@ -35,3 +30,31 @@ ALLOW_REGISTRATION=false
+# El valor por defecto es api:3008 (servicio interno de Compose en la red de Docker).
+# Descomenta solo si el backend corre en otra máquina, puerto o hostname.
+# API_UPSTREAM=api:3008
+
+# ── WebAuthn / Login con huella ──────────────────────────────────────────────
+# RP ID: dominio (sin esquema ni puerto) desde el que se sirve la app. Debe
+# coincidir exactamente con el dominio real del navegador. En local, déjalo
+# como 'localhost'. En producción, usa el dominio público (ej: fin.midominio.es).
+WEBAUTHN_RP_ID=localhost
+# Origen(es) completos (con esquema) desde los que se sirve la app, separados
+# por comas. En producción debe ser la URL HTTPS pública.
+WEBAUTHN_ORIGIN=http://localhost:5173,http://localhost:3008
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The WEBAUTHN_ORIGIN key should go before the WEBAUTHN_RP_ID key

(UnorderedKey)